assets/pages/regular-contributions/components/regularContributionsPayment.jsx

```diff
@@ -159,7 +159,10 @@ function mapStateToProps(state) {
     isTestUser: state.page.user.isTestUser || false,
     isPostDeploymentTestUser: state.page.user.isPostDeploymentTestUser,
     email: state.page.user.email,
-    disable: emptyInputField(state.page.user.firstName) || emptyInputField(state.page.user.lastName),
+    disable:
+      emptyInputField(state.page.user.firstName)
+      || emptyInputField(state.page.user.lastName)
+      || !validateEmailAddress(state.page.user.email),
```

Comment:
Do we have an error message to explain this problem?

Reply:
This will be addressed in a future PR, which will be put out before we start pushing to the new endpoint (`!validateEmailAddress(state.page.user.email)` is always `false` when the user is signed in so this doesn't affect the regular flow)